### ISSUE TITLE
Notifications on all displays

### ISSUE THREAD

**Issue by fullphat** [fullphat/snarl#28]
SnarlOSD doesn't have an option to display the same notification on all displays.

**Comment by jafin** [comment]
Does this issue also imply changing notification to show on different screen?  I've loaded 5 beta 2, and noticed it shows only on the right screen, whereas I cannot locate an option to configure it to show on the left screen.

**Comment by fullphat** [comment]
Yes, that functionality is already in Beta 3.  I'm also looking to add a "display on all screens" option but that would need some reworking of the OSD engine.
